docs/subsystems/analytics.md

```diff
@@ -20,64 +20,63 @@ effectively modify the system.
 
 There are three main components:
 
-- models: The UserCount, StreamCount, RealmCount, and InstallationCount
-  tables (analytics/models.py) collect and store time series data.
-- stat definitions: The CountStat objects in the COUNT_STATS dictionary
-  (analytics/lib/counts.py) define the set of stats Zulip collects.
-- accounting: The FillState table (analytics/models.py) keeps track of what
-  has been collected for which CountStats.
+- models: The `UserCount`, `StreamCount`, `RealmCount`, and `InstallationCount`
+  tables (`analytics/models.py`) collect and store time series data.
+- stat definitions: The `CountStat` objects in the `COUNT_STATS` dictionary
+  (`analytics/lib/counts.py`) define the set of stats Zulip collects.
+- accounting: The `FillState` table (`analytics/models.py`) keeps track of what
+  has been collected for which `CountStat`.
```

Comment:
Github doesn't show these diffs super readably, but these are essentially all just adding \`\` around the names, with no rewriting of the doc content.

Reply:
Sounds good!